beacon-chain/sync/validate_aggregate_proof.go

```diff
@@ -265,7 +265,9 @@ func (s *Service) hasSeenAggregatorIndexEpoch(epoch primitives.Epoch, aggregator
 func (s *Service) setAggregatorIndexEpochSeen(epoch primitives.Epoch, aggregatorIndex primitives.ValidatorIndex) {
 	s.seenAggregatedAttestationLock.Lock()
 	defer s.seenAggregatedAttestationLock.Unlock()
-	b := append(bytesutil.Bytes32(uint64(epoch)), bytesutil.Bytes32(uint64(aggregatorIndex))...)
+	b := make([]byte, 64)
+	copy(b[0:32], bytesutil.Bytes32(uint64(epoch)))
```

Comment:
I assume this code was previously calling `Bytes32` for padding, but since you are already pre-allocating the slice, couldn't you just call copy directly (knowing that the pre-allocated bytes are already zero)? That should save you a couple more allocations right?